vkcs/dc/resource_ip_port_forwarding_test.go

```diff
@@ -0,0 +1,48 @@
+package dc_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/vk-cs/terraform-provider-vkcs/vkcs/internal/acctest"
+)
+
+func TestAccDCIPPortForwarding_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV6ProviderFactories: acctest.AccTestProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: acctest.AccTestRenderConfig(testAccDCIPPortForwardingBasic, map[string]string{
+					"TestAccDCRouterBasic":    acctest.AccTestRenderConfig(testAccDCRouterBasic),
+					"TestAccDCInterfaceBasic": acctest.AccTestRenderConfig(testAccDCInterfaceBasic),
+				}),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("vkcs_dc_ip_port_forwarding.dc_ip_port_forwarding", "name", "tfacc-dc-ip-port-forwarding"),
+					resource.TestCheckResourceAttr("vkcs_dc_ip_port_forwarding.dc_ip_port_forwarding", "description", "tfacc-dc-ip-port-forwarding-description"),
+				),
+			},
+			{
+				ResourceName:      "vkcs_dc_ip_port_forwarding.dc_ip_port_forwarding",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+const testAccDCIPPortForwardingBasic = `
+{{.BaseNetwork}}
+
+{{ .TestAccDCRouterBasic }}
+
+{{ .TestAccDCInterfaceBasic}}
+
+resource "vkcs_dc_ip_port_forwarding" "dc_ip_port_forwarding" {
+	name = "tfacc-dc-ip-port-forwarding"
+	description = "tfacc-dc-ip-port-forwarding-description"
+	dc_interface_id = vkcs_dc_interface.dc_interface.id
+	protocol = "udp"
+    to_destination = "172.17.20.30"
+}
+`
```

Comment:
I would suggest following GCP best practices in acceptance testing for new resources: https://github.com/hashicorp/terraform-provider-google/wiki/Developer-Best-Practices#acceptance-tests. What do you think about it?

Reply:
Rewrote tests using Best Practices. I think, it should be enough to have basic and full-update tests. 
Good topic for discussion is larger resources, that take a lot of time to test. Perhaps, one full-update test will be enough, just to save some time on basic test. On the other hand, it is good idea to test bare-bones functionality. We should discuss this in future